github/resource_github_organization_project.go

```diff
@@ -0,0 +1,116 @@
+package github
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	"github.com/google/go-github/github"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceGithubOrganizationProject() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceGithubOrganizationProjectCreate,
+		Read:   resourceGithubOrganizationProjectRead,
+		Update: resourceGithubOrganizationProjectUpdate,
+		Delete: resourceGithubOrganizationProjectDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"body": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"url": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
```

Comment:
How do you feel about coverage of the remaining fields which are available through `PATCH`?
https://developer.github.com/v3/projects/#update-a-project
specifically `state`, `organization_permission` and `public`?

Reply:
:thinking: It seems that the library that we are using does now support these fields yet?
- There is a way to set the `state` via [`ProjectOptions`](https://godoc.org/github.com/google/go-github/github#ProjectOptions) but the value of the `state` is not included in [`Project`](https://godoc.org/github.com/google/go-github/github#Project) so we can't get the current state of the project via the library.
- For `organization_permission`, is seems that there is an issue opened and being worked on currently https://github.com/google/go-github/issues/884
- For `public`, is it not available via the go-github library yet.

Reply:
Ah, understood. Thanks for the detailed answer & research. We shall leave it out for the initial implementation then.